doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc

```diff
@@ -72,15 +72,27 @@ In this example:
 The RH-SSO server converts the user names to lower case. Therefore, after integration with RH-SSO, your user name will appear in lower case in {PRODUCT}. If you have user names in upper case hard coded in business processes, the application might not be able to identify the upper case user.
 ====
 +
-. Navigate to the `_EAP_HOME_/standalone/configuration` directory in your {EAP} installation.
-  Locate the Elytron and undertow subsystem configurations in the `standalone.xml` and `standalone-full.xml` files and enable JACC. For example:
+. To enable the JACC manually in the `standalone.xml` or in the file where Elytron is installed, do any of the following tasks:
+
+* Elytron subsystem provides a built-in policy provider based on JACC specification. To create the policy provider, enter the following command in the management command-line interface (CLI) of {EAP}:
++
+--
+[source]
+----
+/subsystem=elytron/policy=jacc:add(jacc-policy={})
```

Comment:
```suggestion
/subsystem=elytron/policy=jacc:add(jacc-policy={})
/subsystem=undertow/application-security-domain=other:remove
/subsystem=undertow/application-security-domain=other:add(http-authentication-factory=keycloak-http-authentication,enable-jacc=true)
```
I would add also these two lines. This will do exactly the same as the manual editing of the standalone.xml/standlone-full.xml file in the second task.